src/sqlitetablemodel.cpp

```diff
@@ -390,8 +411,18 @@ QVariant SqliteTableModel::data(const QModelIndex &index, int role) const
                   .arg(QKeySequence(Qt::CTRL).toString(QKeySequence::NativeText));
         else
             return QString();
+    } else if (role == Qt::TextAlignmentRole) {
+        QString value = cached_row->at(column);
+        lock.unlock();
+        QVariant condFormat = getMatchingCondFormat(index.column(), value, role);
+        if (condFormat.isValid())
+            return condFormat;
+        bool isNumber;
+        value.toDouble(&isNumber);
+        return isNumber ? Qt::AlignRight : Qt::AlignLeft;
```

Comment:
Can you add vertical alignment here? Like so:
```c++
return static_cast<int>((isNumber ? Qt::AlignRight : Qt::AlignLeft) | Qt::AlignVCenter);
```
Same for the case where there is a conditional format. At least on my system it looks kind of weird without this because Qt set the row height a bit to high.

Reply:
Or what about AlignBottom? That is what LibreOffice does by default.

Reply:
On the other hand, LibreOffice centers vertically when the height of the text and the cell are the default. I'll use your proposal, since it was the alignment used before this change.